cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/transformers/AbstractInputsTransformer.java

```diff
@@ -93,7 +93,10 @@ private Input createPropInput(Map.Entry<String, Map<String, Serializable>> entry
 
         if (privateInput && !defaultSpecified) {
             throw new RuntimeException(
-                    "Input: " + inputName + " is private but no default value was specified");
+                    "Input: '" + inputName + "' is private but no default value was specified");
```

Comment:
I think we should merge these two if branches into one condition

the defaultSpecified should cover ~ StringUtils.isNotEmpty(proper.get(key))

currently
```
    - input1:
        default: None
        private: true
        required: true
```

case is not covered - please add a test